src/Command/TwoStepVerificationCommand.php

```diff
@@ -21,6 +23,26 @@
 
 class TwoStepVerificationCommand extends ContainerAwareCommand
 {
+    /**
+     * @var ?Helper
+     */
+    private $helper;
+
+    /**
+     * @var ?UserManagerInterface
+     */
+    private $userManager;
+
+    public function __construct(
+        ?string $name,
+        ?Helper $helper = null,
+        UserManagerInterface $userManager = null
```

Comment:
why not `?UserManagerInterface` like `?Helper` ?

Reply:
Sorry, that is a mistake